src/main/java/Duke.java

```diff
@@ -1,10 +1,283 @@
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.*;
+
 public class Duke {
+    private boolean isActive;
+    private List<Task> myList;
+
+    public Duke() {
+        this.isActive = true;
+        this.myList = new ArrayList<>();
+    }
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Duke ft = new Duke();
+        ft.init();
+    }
+    public void init() {
```

Comment:
A more descriptive name might be `greetUser()` as its specific to the context of the class and the client knows what action is triggered by calling this method rather than if it was simply `init`